src/util.rs

```diff
@@ -264,10 +264,10 @@ impl U32Ext for u32 {
     fn to_ty(&self) -> Result<Ident> {
         Ok(match *self {
             1 => Ident::new("bool"),
-            2...8 => Ident::new("u8"),
-            9...16 => Ident::new("u16"),
-            17...32 => Ident::new("u32"),
-            33...64 => Ident::new("u64"),
+            2..=8 => Ident::new("u8"),
```

Comment:
Would you mind moving out all those unrelated cleanups into a separate PR?

Reply:
Done #307